components/metrics/labels.go

```diff
@@ -0,0 +1,54 @@
+package metrics
+
+import (
+	"context"
+
+	"github.com/ThreeDotsLabs/watermill/message"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type labelKey string
+
+const (
+	labelKeyHandlerName    = "handler_name"
```

Comment:
i think it is good idea to have typed (`labelKey`) consts here

Reply:
I've decided not to have typed label keys in metrics, just strings. These are keys for prometheus labels, which are actually strings. Typed strings are needed when dealing with context.Context, here it was overkill, I guess

Reply:
e93109c91a43297193a6cc47de0144aceba97944